pkg/client/clientcmd/overrides.go

```diff
@@ -141,7 +142,7 @@ func BindAuthInfoFlags(authInfo *clientcmdapi.AuthInfo, flags *pflag.FlagSet, fl
 // BindClusterFlags is a convenience method to bind the specified flags to their associated variables
 func BindClusterFlags(clusterInfo *clientcmdapi.Cluster, flags *pflag.FlagSet, flagNames ClusterOverrideFlags) {
 	flags.StringVarP(&clusterInfo.Server, flagNames.APIServer, flagNames.APIServerShort, "", "The address and port of the Kubernetes API server")
-	flags.StringVar(&clusterInfo.APIVersion, flagNames.APIVersion, "", "The API version to use when talking to the server")
+	flags.StringVar(&clusterInfo.APIVersion, flagNames.APIVersion, latest.Version, "The API version to use when talking to the server")
```

Comment:
Its picking up latest.Version. I changed this just to be explicit.
Let me know if you want me to revert.